### PR TITLE
Feature/pipeline fail on uncommitted files

### LINKED ISSUE #2109: release script throws error "uncommitted changes"
Changed the folder path of svg files at materials and wanted to release a braeking change version of materials. 

```
 423 passed (9m 05s)
lerna notice cli v3.20.2
lerna info versioning independent
lerna info Looking for changed packages since @axa-ch/button-link@5.0.11
lerna info ignoring diff in paths matching [
lerna info ignoring diff in paths matching   '**/*.test.js',
lerna info ignoring diff in paths matching   '**/node_modules/**',
lerna info ignoring diff in paths matching   '**/package-lock.json',
lerna info ignoring diff in paths matching   '**/story.js',
lerna info ignoring diff in paths matching   '**/README.md',
lerna info ignoring diff in paths matching   '**/CHANGELOG.js'
lerna info ignoring diff in paths matching ]
lerna ERR! EUNCOMMIT Working tree has uncommitted changes, please commit or remove the following changes before continuing:
lerna ERR! EUNCOMMIT  M package-lock.json
lerna ERR! EUNCOMMIT  M src/components/10-atoms/button-link/package-lock.json
lerna ERR! EUNCOMMIT  M src/components/10-atoms/button/package-lock.json
lerna ERR! EUNCOMMIT  M src/components/10-atoms/icon/package-lock.json
lerna ERR! EUNCOMMIT  M src/components/10-atoms/input-file/package-lock.json
lerna ERR! EUNCOMMIT  M src/components/10-atoms/input-text/package-lock.json
lerna ERR! EUNCOMMIT  M src/components/10-atoms/link/package-lock.json
lerna ERR! EUNCOMMIT  M src/components/20-molecules/cookie-disclaimer/package-lock.json
lerna ERR! EUNCOMMIT  M src/components/20-molecules/datepicker/package-lock.json
lerna ERR! EUNCOMMIT  M src/components/20-molecules/dropdown/package-lock.json
lerna ERR! EUNCOMMIT  M src/components/20-molecules/file-upload/package-lock.json
lerna ERR! EUNCOMMIT  M src/components/20-molecules/popup/package-lock.json
lerna ERR! EUNCOMMIT  M src/components/20-molecules/stepper/package-lock.json
lerna ERR! EUNCOMMIT  M src/components/20-molecules/top-content-bar/package-lock.json
lerna ERR! EUNCOMMIT  M src/components/30-organisms/footer/package-lock.json
```

### PR DESCRIPTION
**Pull request by raphaellueckl**
Fixes #2109

# Merge Checklist:
- [x] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [x] Tests are sufficient.
- [x] Modifications still work in IE.
- [x] Documentation is up to date.
- [x] Changelog is up to date.
- [x] Typescript typings for properties are up to date.
- [x] Designers approved changes or no approval needed.
- [x] Dependencies to other components still work.

### CODE REVIEW

**Review by danyball — CHANGES_REQUESTED** [review]
i dont think this check can help. lerna is changing the lock files i think

**Comment by raphaellueckl** [comment]
What do you mean? Lerna is also installing all of our components before testing it. And often, after `npm i`, there are some package-lock files that are not being committed. This makes sure that this does not happen.

**Comment by danyball** [comment]
> What do you mean? Lerna is also installing all of our components before testing it. And often, after `npm i`, there are some package-lock files that are not being committed. This makes sure that this does not happen.

i have developed a new feature at materials. The version at package.json has been changed by lerna. (i choose major release). And after that, lerne updated all the package.jsons of components who are using materials.

File: package.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
     "test-testcafe-headless": "TEST_HOST_STORYBOOK_URL=http://localhost:6006 testcafe 'chrome:headless' ./**/ui.test.js",
     "new": "node scripts/create-a-m-o/index.js",
     "deploy-branch": "npm run build-storybook && SOURCE_BRANCH_NAME=\"$(git rev-parse --abbrev-ref HEAD)\" bash scripts/branch-deployment.sh && SOURCE_BRANCH_NAME=\"$(git rev-parse --abbrev-ref HEAD)\" bash scripts/branch-deployment-notifier.sh",
-    "link-components": "sh scripts/npm-link-components.sh"
+    "link-components": "sh scripts/npm-link-components.sh",
+    "check-for-uncommitted-files": "git status && git diff-index --quiet HEAD --"
```
Comment (danyball) [inline package.json:28]: do you need the two "--" at the end?

**Review by danyball — APPROVED** [review]

**Pull request merged**